src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2888,8 +2888,8 @@ public void setStereoscopy(String stereoscopy) {
 		SBSL,
 		SBSR,
 		HSBSL,
-		OUL,
-		OUR,
+		ABL,
+		ABR,
```

Comment:
@valib I have no idea what these codes mean. Shouldn't ```OUx``` be used anymore/isn't it valid?